package/kedro_viz/server.py

```diff
@@ -229,8 +244,29 @@ def _call_viz(host=None, port=None, browser=None, load_file=None, save_file=None
                 sys.exit(1)
     else:
         try:
+            # Kedro 0.15.0+
+            from kedro.context import KedroContextError
+
+            context = get_project_context()
+            try:
+                # Kedro 0.15.2+
+                pipeline = context._get_pipeline(  # pylint: disable=protected-access
+                    name=pipeline_name
+                )
+            except NotImplementedError:
+                # Kedro 0.15.0 or 0.15.1
+                if pipeline_name:
+                    raise KedroCliError(
+                        "`--pipeline` flag was provided, but there is no multiple pipelines."
```

Comment:
Maybe
```suggestion
                        "`--pipeline` flag was provided, but there is not support for multiple pipelines for your Kedro version."
```
or sth along those lines?

Reply:
Thanks! I will change the message to be more general (to cover KeyError and Not ImplementedError).